examples/swe_agent/main.ipynb

```diff
@@ -0,0 +1,333 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example with SWE-agent\n",
+    "\n",
+    "SWE-agent is an agent designed for solving github issues.\n",
```

Comment:
Give a more specific, official introduction for SWE-agent, for example, what does "SWE" means？

Reply:
Added more description